.mypy.ini

```diff
@@ -1,24 +1,45 @@
 [mypy]
-warn_unused_configs = True
-strict = True
+files = aiohttp, examples, tests
+check_untyped_defs = True
+follow_imports_for_stubs = True
+#disallow_any_decorated = True
```

Comment:
These 2 commented out, we should consider in the future, but will need a bunch of errors to be cleaned up or ignored first.